snapcraft/libraries.py

```diff
@@ -48,3 +56,40 @@ def _extract_ld_library_paths(ld_conf_file):
                 paths.extend(path_delimiters.split(line))
 
     return paths
+
+
+_libraries = None
+
+
+def _get_system_libs():
+    global _libraries
+    if _libraries:
+        return _libraries
+
+    release = platform.linux_distribution()[1]
+    lib_path = os.path.join(common.get_librariesdir(), release)
+
+    if not os.path.exists(lib_path):
+        logger.warning('No libraries to exclude from this release')
```

Comment:
From the text in this message, it's not clear why it is a warning. How can affect the execution that there are no libraries to exclude?